CHANGES.md

```diff
@@ -17,6 +17,9 @@ Release 4.2.1
 
 Features
 --------
+* [#529](https://github.com/java-native-access/jna/pull/529): Added new files `com.sun.jna.platform.win32.Mpr`, `com.sun.jna.platform.win32.LmShare`, and `com.sun.jna.platform.win32.Winnetwk` [@amarcionek](https://github.com/amarcionek).
```

Comment:
No need to say "new files".
Missing dash before your name, see other lines and correct the one from @alexvsimon below.